### PR TITLE
Feature/scrum 24

### PR DESCRIPTION
**Pull request by Michaljarek**

### CODE REVIEW

File: rick-morty-app-client/src/components/atoms/Avatar/Avatar.module.scss [hunk 1]
```diff
@@ -1,4 +1,5 @@
-@use "styles/mixins" as *;
+@use "../../../styles/mixins" as *;
+@use "../../../styles/variables" as *;
```
Comment (AlicjaSzot) [inline rick-morty-app-client/src/components/atoms/Avatar/Avatar.module.scss:3]: Remove "../../../" from line 1 and 2

File: rick-morty-app-client/src/components/molecules/StatusBadge/StatusBadge.module.scss [hunk 2]
```diff
@@ -0,0 +1,29 @@
+@use "../../../styles/mixins" as *;
+@use "../../../styles/variables" as *;
```
Comment (AlicjaSzot) [inline rick-morty-app-client/src/components/molecules/StatusBadge/StatusBadge.module.scss]: It should be:
 "@use "styles/mixins" as *;
 @use "styles/variables" as *;

File: rick-morty-app-client/src/components/molecules/StatusBadge/StatusBadge.tsx
Comment (AlicjaSzot) [inline rick-morty-app-client/src/components/molecules/StatusBadge/StatusBadge.tsx:1]: In folder "config" > folder "webpack" find "webpack.config.js" > search "alias" (313 line) and in this part add path for your types folder. it should be like this: 
"@types": path.resolve(__dirname, "src/types"),". 
And Next in your StatusBadge.tsx file change path for import CharacterStatus to "import { CharacterStatus } from "types/CharactersTypes";"

Reszta nie wiem co sie dzieje, ale jak działa to ok.

Comment (Michaljarek) [reply rick-morty-app-client/src/components/molecules/StatusBadge/StatusBadge.tsx:1]: Działa

File: rick-morty-app-client/src/components/molecules/StatusBadge/StatusBadge.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import styles from "./StatusBadge.module.scss";
+import { CharacterStatus } from "../../../types/CharactersTypes";
```
Comment (Pablo2904) [inline rick-morty-app-client/src/components/molecules/StatusBadge/StatusBadge.tsx]: remove ../../../

**Review by Pablo2904 — CHANGES_REQUESTED** [review]

**Review by Pablo2904 — APPROVED** [review]

**Pull request merged**